UnityProject/Assets/Scripts/Core/Editor/GenerateSpriteSO.cs

```diff
@@ -123,6 +124,12 @@ public static void Reset()
 		AssetDatabase.SaveAssets();
 	}
 
+
+	public static double GetRandomNumberInRange(double minNumber, double maxNumber)
+	{
+		return new System.Random().NextDouble() * (maxNumber - minNumber) + minNumber;
+	}
```

Comment:
wut, that does not do what the function is called

Reply:
is this for generating IDs again? Can't you just use C#'s UUID generator?

Reply:
this is for just testing